pages/consultancy.md

```diff
@@ -1,64 +1,64 @@
 ---
-title: "Research Software Engineering (RSE)"
+title: "Consultancy Support for Research"
 permalink: "/consulting/"
 breadcrumb: true
 ---
 
-The Research Software Engineering (RSE) service within Research Computing offers specialist software development consultancy, code development expertise and access to tools, platforms and training to research projects at the University of Leeds.
+Our Research Computing team at the University of Leeds, delivered through our dedicated Research Software Engineers (RSEs), offers a comprehensive range of expert services designed to support researchers throughout the entire lifecycle of their projects. You can meet members of the team via the [Team page]({{ site.baseurl }}/about/team/).
```

Comment:
Maybe rephrase to be "Our Research Computing **service**... delivered through our RSEs" - just the "team" paired with "delivered" sounds a bit clunky to me?